src/pymovements/gaze/io.py

```diff
@@ -296,16 +300,16 @@ def from_asc(
 
     Returns
     -------
-    GazeDataFrame
-        The gaze data frame read from the asc file.
+    tuple[GazeDataFrame, dict[str, Any]]
+        The gaze data frame and a metadata dictionary read from the asc file.
 
     Examples
     --------
     Let's assume we have an EyeLink asc file stored at `tests/files/eyelink_monocular_example.asc`.
     We can then load the data into a ``GazeDataFrame``:
 
     >>> from pymovements.gaze.io import from_asc
-    >>> gaze = from_asc(file='tests/files/eyelink_monocular_example.asc', patterns='eyelink')
```

Comment:
why was patterns='eyelink' deleted?

Reply:
Because it's the default value anyway, and since this is part of the docstring, I wanted to make the example as simple as possible. I can revert it if you prefer.